xml/net_samba.xml

```diff
@@ -778,13 +730,74 @@
     configuration.
    </para>
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
+   </para>
+   <note>
```

Comment:
First saying it is deprecated and then having a note saying it is still supported in some cases come a bit as a surprise... maybe you could replace 'supported' by s.th. else?